apps/wallet-ui/src/app/[ecosystem]/login/page.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+import { client } from "@/lib/client";
+import { useTheme } from "next-themes";
+import { ConnectEmbed } from "thirdweb/react";
+import { ecosystemWallet } from "thirdweb/wallets";
+import { generatePayload, isLoggedIn, login, logout } from "./auth";
+
+export default function Page({ params }: { params: { ecosystem: string } }) {
+  const { theme } = useTheme();
+
+  return (
+    <ConnectEmbed
+      theme={theme as "light" | "dark"}
+      autoConnect={true}
+      wallets={[ecosystemWallet(`ecosystem.${params.ecosystem}`)]}
+      client={client}
+      auth={{
+        getLoginPayload: async (params) => generatePayload(params),
+        doLogin: async (params) => {
+          await login(params, `wallet/${params.payload.address}`);
+        },
+        isLoggedIn: async () => false,
```

Comment:
wat?

Reply:
wat is right, this was for testing it snuck through